app/isotopomer/__init__.py

```diff
@@ -462,7 +461,7 @@ def extract_site_dictionary_from_dash_triggers(states):
     for angle in ["alpha", "beta", "gamma"]:
         if (
             "shielding_symmetric" in site.keys()
```

Comment:
It might be optimum to put the condition `if "shielding_symmetric" in site.keys()` outside the for loop. This statement has nothing to do with the loop